google/datalab/ml/_summary.py

```diff
@@ -95,10 +95,9 @@ def get_events(self, event_names):
           events (i.e. 'loss' under trains_set/, and 'loss' under eval_set/.)
     """
 
-    if sys.version_info.major > 2:
-      basestring = (str, bytes)  # for python 3 compatibility
-
-    event_names = [event_names] if isinstance(event_names, basestring) else event_names
+    if ((sys.version_info.major > 2 and isinstance(event_names, str)) or
+        (sys.version_info.major <= 2 and isinstance(event_names, basestring))):
```

Comment:
Good idea. I am a bit hesitate to introduce a dependency on six now. Will open an issue for that.